### PR TITLE
890: jdk.json mapping modules and some files to the wrong area

### PR DESCRIPTION
**Pull request by magicus**
Update jdk.json mailing list rules.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Change must be properly reviewed

### Issue
 * [SKARA-890](https://bugs.openjdk.java.net/browse/SKARA-890): jdk.json mapping modules and some files to the wrong area

### Reviewers
 * [Alan Bateman](https://openjdk.java.net/census#alanb) (@AlanBateman - no project role)
 * [Erik Helin](https://openjdk.java.net/census#ehelin) (@edvbld - **Reviewer**)

### Download
`$ git fetch https://git.openjdk.java.net/skara pull/1063/head:pull/1063`
`$ git checkout pull/1063`

### CODE REVIEW

**Comment by edvbld** [comment]
@AlanBateman would you mind having a look at the changes? Thanks! :pray:

File: config/mailinglist/rules/jdk.json [hunk 1]
```diff
@@ -152,83 +136,23 @@
...
-            "src/java.base/windows/classes/java/io/",
-            "src/java.base/windows/classes/java/lang/",
-            "src/java.base/windows/classes/jdk/internal/loader/",
-            "src/java.base/windows/classes/sun/util",
-            "src/java.base/windows/native/common",
-            "src/java.base/windows/native/include",
-            "src/java.base/windows/native/libjava/",
-            "src/java.base/windows/native/libjimage",
-            "src/java.base/windows/native/libjli/",
+            "src/java.base/",
```
Comment (AlanBateman) [inline config/mailinglist/rules/jdk.json:143]: This looks like core-libs-dev will be mailed for all changes in src/java.base, I don't think we want that as there it a lot of code in java.base that is maintained on other mailing lists.

Comment (magicus) [reply config/mailinglist/rules/jdk.json:143]: You are correct. I misread the statement in the bug report saying: `core-libs-dev cares about everything in src/java.base/*/native/libjava, no need to list specific files` as `core-libs-dev cares about everything in src/java.base/*, no need to list specific files`. My bad; will fix.

**Review by AlanBateman — APPROVED** [review]
Thanks, I think this looks right.

**Review by edvbld — APPROVED** [review]
Looks good

**Comment by edvbld** [comment]
> Thanks, I think this looks right.

Thanks Alan for taking the time to review this!

**Pull request closed without merging**